tests/unit/job_runner_handlers/test_pbs.py

```diff
@@ -40,8 +43,28 @@
                 '#PBS -e cylc-run/chop/log/job/1/axe/01/job.err',
                 '#PBS -l walltime=180',
             ],
+            id='basic'
+        ),
+        pytest.param(
+            {
+                'directives': {},
+                'execution_time_limit': 180,
+                'job_file_path': 'cylc-run/chop/log/job/1/axe/01/job',
+                'workflow_name': 'chop',
+                'task_id': VERY_LONG_STR,
+                'platform': {
+                    'job runner': 'pbs',
+                }
+            },
+            [
+                f'#PBS -N None.{VERY_LONG_STR[:231]}',
```

Comment:
```suggestion
                f'#PBS -N None.{VERY_LONG_STR[:PBSHandler.JOB_NAME_LEN_MAX - 5]}',
```

Reply:
Best to add these changes locally because I can't suggest the `import PBSHandler` line for you

Reply:
I've already committed it?